Reneiski_Homework1/Zavod/Person.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WorkerNamespace
+{
+    public class Person
+    {
+        private string firstName;
+        private string lastName;
+        private int age;
+
+        public Person(string firstName, string lastName, int age)
+        {
+            this.firstName = firstName;
+            this.lastName = lastName;
+            this.age = age;
+        }
+
+        public virtual void ShowPersonInfo()
+        {
+            Console.WriteLine(firstName);
+            Console.WriteLine(lastName);
+            Console.WriteLine(age);
+        }
+
+        public virtual ref string ShowPersonLastName()
+        {
+            return ref lastName;
+        }
+
+        public virtual ref string ShowPersonFirstName()
```

Comment:
it' better to rename all methods which return smth to smth like GetPersonFirstName because you actually return string rather than show. Please correct here and across all files of Lecture5/Lecture6

Reply:
Fixed